### DISCUSSION TITLE
RUNES 0 (GENESIS RUNES) AS GAS OF RUNES PROTOCOL

### DISCUSSION THREAD

**Discussion by fiiph** [ordinals/ord#3111] [Ideas]
ORDINALS/RUNES make bitcoin network much more interesting than before! Thanks for that.
While unlimited/infinite minting is boring and makes bitcoin network busy to use.
How about define RUNES 0 (GENESIS RUNES) as a FT which will be used as the GAS token for all future RUNES etching/minting? Except RUNES 0 itself, every RUNES etching/minting MUST burn one RUNES 0 token.
And then RUNES 0 becomes a deflation token.

**Comment by cbspears** [comment]
Dude. Bitcoin is the gas of runes.

**Reply by raphjaph** [reply]
🤯

**Comment by cliobitcoinbank** [comment]
Let's just remove the ability to name any rune bitcoin, consider bitcoin itself as a rune, make bitcoin the official gas token of runes.

**Comment by raphjaph** [comment]
Please transfer this discussion here: https://github.com/casey/runestone
